lib/mix/lib/mix/tasks/test.ex

```diff
@@ -146,6 +146,11 @@ defmodule Mix.Tasks.Test do
 
       mix test test/some/particular/file_test.exs
 
+  Tests in umbrella apps can specify the full path including `apps/my_app/`
```

Comment:
In order to run all tests in an umbrella, I need to do this though:

    mix test apps/my_app/test

Correct? I.e. the "test" directory is required.

Reply:
The idea with that "including" note was to highlight the difference between an umbrella and a regular app, the delta being only those two folders. `test` is required in both so isn't a difference. `test` is included in the actual example for both so it should hopefully be clear that it is required.